site/gatsby-site/src/components/styles/Docs.js

```diff
@@ -38,7 +38,6 @@ export const Edit = styled.div`
 `;
 
 export const StyledMainWrapper = styled.div`
-  max-width: 800px;
```

Comment:
This makes plain textual pages, like `/about`, [too wide to read comfortably](https://en.wikipedia.org/wiki/Line_length). I think it would be better to unset `max-width` for the blog post listing specifically.

![image](https://user-images.githubusercontent.com/25443411/189987987-b5d39518-4dc5-4e8b-89da-46f81a96bc09.png)

Reply:
Otherwise, it looks good!

Reply:
@lmcnulty fixed! Ready for another round